docs/tutorial/typesfuns.rst

```diff
@@ -149,10 +149,13 @@ Idris prompt:
 
 .. note::
 
-    Idris automatically desugars the ``Nat`` representation into a
-    more human readable format. The result of ``plus (S (S Z)) (S (S Z))``
-    is actually ``(S (S (S (S Z))))`` which is the Integer 4. This can be
-    checked at the Idris prompt:
+   When displaying an element of ``Nat``,
+   such as ``(S (S (S (S Z))))``, for a user.
+   Idris displays it as 4 -- a more human readable format.
```

Comment:
While you're at it, how about "Idris displays it as ``` ``4`` ```, which is easier for humans to read."  I think this phrasing is easier for humans to read (har har), and it's also referring to the example rather than to the format that lies implicit in the example.  Note that this comment is me being exceedingly picky :-)